jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -454,16 +498,28 @@ public JibContainerBuilder toContainerBuilder() throws IOException {
     // Add others to layer configuration
     for (Path path : addedOthers) {
       if (Files.isDirectory(path)) {
-        layerConfigurationsBuilder.addDirectoryContents(
-            LayerType.EXTRA_FILES, path, path1 -> true, appRoot.resolve(othersDestination));
+        addDirectoryContentsToLayer(
+            layerBuilders,
+            LayerType.EXTRA_FILES,
+            path,
+            path1 -> true,
```

Comment:
I'd go with `ignored -> true`.

Reply:
Don't forget this.

Reply:
Oh weird, I definitely changed this. I guess I hit undo too many times before committing or something.